esrally/driver/driver.py

```diff
@@ -26,11 +26,13 @@
 import queue
 import threading
 import time
+from enum import Enum
 
 import thespian.actors
 
 from esrally import actor, config, exceptions, metrics, track, client, paths, PROGRAM_NAME, telemetry
 from esrally.driver import runner, scheduler
+from esrally.track.loader import TrackProcessorRegistry, load_track_plugins
```

Comment:
Can we expose these two in the `__init__.py` of `esrally.track` so that the (low-level) `loader` submodule is not directly used?